vhost-user-backend/src/event_loop.rs

```diff
@@ -58,7 +95,11 @@ pub type VringEpollResult<T> = std::result::Result<T, VringEpollError>;
 /// - remove registered file descriptors from the epoll fd
 /// - run the event loop to handle pending events on the epoll fd
 pub struct VringEpollHandler<T: VhostUserBackend> {
```

Comment:
Ditto, should we rename this to `VringPollHandler`?

If it's a lot of renaming, I'd do that in a separate commit where we just rename structs (before or after using `mio`) to simplify the review.